src/pages/[...slug].tsx

```diff
@@ -56,28 +56,41 @@ export async function getStaticProps(
   const addResourceToCollection = nodeMeta[type]?.additionalNode
 
   /** Check for is collection variable and determines if it is a collection or a single resource */
-
   if (!isCollection) {
     node = await drupalClient.getResourceFromContext<NodeTypes>(path, context, {
-      params: nodeMeta[type].params.getQueryObject(),
+      params: {
+        ...nodeMeta[type].params.getQueryObject(),
+      },
     })
   }
+
   if (isCollection) {
     node = await drupalClient.getResourceCollectionFromContext<NodeTypes>(
       type,
       context,
-      { params: nodeMeta[type].params.getQueryObject() }
+      {
+        params: {  
+          'filter[drupal_internal__nid][value]': path.entity.id,
+          ...nodeMeta[type].params.getQueryObject(),
```

Comment:
filtering by `path.entity.id` fixes the issue but is there a better way to do this?